frontend/src/templates/index.mustache

```diff
@@ -11,7 +11,7 @@
     <meta name="viewport" content="width=device-width">
     <link rel="localization" href="/static/locales/{locale}/app.ftl">
 
-    <link rel="canonical" href="https://testpilot.firefox.com/">
+    <link rel="canonical" href="https://testpilot.firefox.com/{{{ canonical_path }}}">
```

Comment:
Locally, when I open the file from the filesystem I see the correct path here, but if I curl (for example) `http://testpilot.dev:8000/experiments/min-vid` I only see `https://testpilot.firefox.com/`. I can't tell if that's a side effect of the dev server or not.